org.freecadweb.FreeCAD.yaml

```diff
@@ -503,11 +504,10 @@ modules:
           - exec /app/freecad/bin/FreeCAD -u ${XDG_CONFIG_HOME}/FreeCAD/user.conf -s ${XDG_CONFIG_HOME}/FreeCAD/system.cfg "$@"
       - type: git
         url: https://github.com/FreeCAD/FreeCAD.git
-        # branch: FreeCAD-0-18
-        commit: 783e1346c7a078f5265c3f0c6a927d09a8c1fe70
-      - type: patch
-        path: patches/freecad-pyside-5.14.patch
+        # tag: 0.19_pre
+        # commit #22492 as built from master
+        commit: fe3f1b5
```

Comment:
I guess it works either way, but I would prefer the full hash for consistency.

Reply:
done